### PR TITLE
Dg add ajax

### PR DESCRIPTION
**Pull request by dgarcia13-99**

### CODE REVIEW

File: app/controllers/follow_requests_controller.rb [hunk 1]
```diff
@@ -22,12 +22,13 @@ def edit
   # POST /follow_requests or /follow_requests.json
   def create
     @follow_request = FollowRequest.new(follow_request_params)
-    @follow_request.sender = current_user
+    @follow_request.sender_id = current_user.id
```
Comment (hatcheta1) [inline app/controllers/follow_requests_controller.rb:25]: nice change to ```sender_id```

File: app/views/comments/_comment.html.erb [hunk 2]
```diff
@@ -11,11 +11,11 @@
     </div>
     <div>
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted", remote: true do %>
           <i class="fas fa-edit fa-fw"></i>
         <% end %>
 
-        <%= link_to comment, data: { turbo_method: :delete }, class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to comment, method: :delete, class: "btn btn-link btn-sm text-muted", remote: true do %>
```
Comment (hatcheta1) [inline app/views/comments/_comment.html.erb:18]: nice refactoring!

File: app/views/comments/create.js.erb
Comment (hatcheta1) [inline app/views/comments/create.js.erb:1]: nice animation!

File: app/views/comments/update.js.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment.photo) %>_<%= dom_id(@comment) %>_form").replaceWith("<%= j(render @comment) %>");
```
Comment (hatcheta1) [inline app/views/comments/update.js.erb:1]: good and concise!

File: app/views/follow_requests/destroy.js.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+//Update follow/un-follow/un-request button
+$(".sender_<%= @follow_request.sender.id %>_recipient_<%= @follow_request.recipient.id %>_follow_unfollow")
+  .replaceWith("<%= j(render "follow_requests/follow_unfollow", sender: @follow_request.sender, recipient: @follow_request.recipient) %>");
```
Comment (hatcheta1) [inline app/views/follow_requests/destroy.js.erb:3]: good use of ```replaceWith```!

**Review by hatcheta1 — APPROVED** [review]
looks great!

**Review by aldoportillo — APPROVED** [review]
Great work.

**Pull request merged**